modules/caddyhttp/reverseproxy/caddyfile.go

```diff
@@ -270,6 +287,60 @@ func (h *Handler) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 				}
 				h.LoadBalancing.TryInterval = caddy.Duration(dur)
 
+			case "lb_retry_match":
+				// this code is essentially copied from MatchNot
```

Comment:
I wonder if we can reuse that code. That would certainly be preferable if it's pretty straightforward.